test/e2e_node/kubelet_perf_test.go

```diff
@@ -0,0 +1,277 @@
+/*
```

Comment:
Let's rename this file to `resource_usage_test` to be more accurate, or maybe `daemon_resource_usage_test`.